package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "svg2react-icon",
-  "version": "3.0.1",
+  "version": "4.0.1",
```

Comment:
Maybe instead of making a breaking change, we will just expose `outputDir` as an external configuration and set its default to `components/`?

Whenever it's possible to avoid breaking changes, I would :P

Reply:
Yeah, I wanted to do that at first.
But the API will be really unintuitive.
The CLI tool already accepts an `outputDir` name, and on top of that adds the `/component/` folder.
How would a new option work?
maybe add a `subDir`?

Reply:
Hmm, what about `outputDirPrefix`, `baseOutputDir` or just `baseDir`?

Reply:
I actually kind of misled with this PR's name.
This isn't a prefix or a base dir, it's a directory added to the given `outputDir`.
Meaning if I run `svg2react-icon inputDir outputDir`, the result will be this structure:
```
outputDir
|
+-- index.ts
|
+-- components
    |
    <icon files>
```

So actually what's happening here is a separation between the icon files themselves and the index file.
I'd rather have them all in the output directory I passed to the CLI.

So maybe some option like `--no-sub-dir` or something like that?

Reply:
Sounds good to me 👍🏻